style.css

```diff
@@ -326,3 +326,259 @@ hr {
   height: 29px;
   left: 76px;
 }
+
+@media only screen and (min-width: 768px) {
+  body {
+    background: url('./Images/two_circle.png');
+    background-color: #191765;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-attachment: fixed;
+  }
+
+  .logo {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  ul {
+    list-style-type: none;
+  }
+
+  a {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .nav-links {
+    justify-content: space-between;
+    align-items: center;
+    gap: 20px;
+  }
+
+  .nav-links li {
+    display: flex;
+    font-size: normal;
+    font-family: 'Roboto', sans-serif;
+    align-items: center;
+    text-align: center;
+  }
+
+  .header-page nav {
+    font-family: 'Roboto', sans-serif;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: #fff;
+  }
+
+  .header-page ul {
+    display: flex;
+    color: #fff;
+  }
+
+  .main-page h1 {
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 90px;
+    line-height: 105px;
+    text-align: center;
+  }
+
+  .main-page h2 {
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 40px;
+    line-height: 47px;
+    text-align: center;
+  }
+
+  .main-page p {
+    display: flex;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    color: #b3bac5;
+    text-align: center;
+    margin-top: 80px;
+    line-height: 32px;
+    height: 155px;
+    width: 447px;
+  }
+
+  .section2 {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 13%;
+    justify-content: center;
+  }
+
+  .break-line {
+    display: flex;
+  }
+
+  .list-heading {
+    color: #9490ff;
+    display: flex;
+  }
+
+  .work-left {
+    align-items: left;
+  }
+
+  .work-right .work-left {
+    display: flex;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    text-align: center;
+  }
+
+  .middle-section h1 {
+    text-align: start;
+    display: flex;
+    align-items: center;
+    font-family: 'Roboto', sans-serif;
+    font-size: 64px;
+    font-style: normal;
+  }
+
+  .middle-section hr {
+    height: 0;
+    left: 0;
+    right: 0;
+    border: 1px solid #646369;
+    justify-content: center;
+    margin-top: 35px;
+  }
+
+  .desktop-us {
+    display: flex;
+    justify-content: center;
+    color: #9390ff;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    margin-top: 40px;
+  }
+
+  .mobile-us {
+    display: none;
+  }
+
+  .work-section {
+    display: flex;
+    margin: 80px;
+  }
+
+  .work-section1 {
+    display: flex;
+    flex-direction: row-reverse;
+    margin: 80px;
+  }
```

Comment:
- [ ] Kindly fix the positions of the elements in the work section to be similar to the Figma template. 👇🏽 

- Figma template

![image](https://user-images.githubusercontent.com/58642949/187029364-e84c8a11-b1ff-48d3-b32e-8b998c07f26c.png)

- Your design
![image](https://user-images.githubusercontent.com/58642949/187029430-a42a9c6d-3088-422a-b4bc-051b3aa7f304.png)